Note: The result of applying this patch is not used.

This patch serves only to check whether the relevant portion of the
export_lite_tarball() function in the publish_tarball.py script has
changed, indicating that we need to update the prune_lite_excluded_dirs()
function in package_chromium.sh.

--- a/build/recipes/recipes/publish_tarball.py	2025-01-15 18:33:24.087269000 -0500
+++ b/build/recipes/recipes/publish_tarball.py	2025-01-17 14:29:18.247254000 -0500
@@ -290,63 +290,9 @@
 
 def export_lite_tarball(api, source_dir, version):
   # Make destructive file operations on the copy of the checkout.
-  with copytree_checkout(api, source_dir) as dest_dir:
-    directories = [
-        'android_webview',
-        'build/linux/debian_bullseye_amd64-sysroot',
-        'build/linux/debian_bullseye_i386-sysroot',
-        'buildtools/reclient',
-        'chrome/android',
-        'chromecast',
-        'ios',
-        'native_client',
-        'native_client_sdk',
-        'third_party/android_platform',
-        'third_party/angle/third_party/VK-GL-CTS',
-        'third_party/apache-linux',
-        'third_party/catapult/third_party/vinn/third_party/v8',
-        'third_party/closure_compiler',
-        'third_party/instrumented_libs',
-        'third_party/llvm',
-        'third_party/llvm-build',
-        'third_party/llvm-build-tools',
-        'third_party/node/linux',
-        'third_party/rust-src',
-        'third_party/rust-toolchain',
-        'third_party/webgl',
-    ]
-    for directory in [
-        'third_party/blink/manual_tests', 'third_party/blink/perf_tests'
-    ]:
-      if api.path.exists(api.path.join(dest_dir, directory)):
-        directories.append(directory)  # pragma: no cover
-
-    for directory in directories:
-      try:
-        api.step('prune %s' % directory, [
-            'find', api.path.join(dest_dir, directory),
-            '-type', 'f,l',
-            '-regextype', 'egrep',
-            '!', '-regex', '.*\.(gn|gni|grd|grdp|isolate|pydeps)(\.[^ /]+)?',
-            '!', '(', '(', '-iname', '*COPYING*',   '-o',
-                           '-iname', '*Copyright*', '-o',
-                           '-iname', '*LICENSE*',
-                      ')',
-                      # Files with these extensions are probably not
-                      # actual license text.
-                      '!', '-iregex', '.*\.(cc|cfg|cpp|h|java|js|json|m|patch|pl|py|rs|sh|sha1|stderr|ts|ya?ml)',
-                 ')',
-            '-delete'
-        ])  # yapf: disable
-      except api.step.StepFailure:  # pragma: no cover
-        # Ignore failures to delete these directories - they can be inspected
-        # later to see whether they have moved to a different location
-        # or deleted in different versions of the codebase.
-        pass
-
-    # Empty directories take up space in the tarball.
-    api.step('prune empty directories',
-             ['find', dest_dir, '-depth', '-type', 'd', '-empty', '-delete'])
+##
+## Relevant code deleted
+##
 
     export_tarball(
         api,
